eclipse-jdt/eclipse.jdt.ui
# Refactor VarCleanUp to jdt.core.manipulation for jdt.ls usage

- create new VarCleanUpCore based on VarCleanUp
- use wrapper of VarCleanUpCore to replace VarCleanUp contents
- fixes #576



## What it does

Refactors VarCleanUp logic to VarCleanUpCore so jdt.ls can use it.

## How to test

There are no functionality changes.  Run CleanUpTest10 to verify logic still works.

## Author checklist
